performance_tests/conftest.py

```
@@ -0,0 +1,107 @@
+    try:
+        # Structlog
+        import structlog
+        structlog.configure(
+            processors=[structlog.processors.JSONRenderer()],
+            wrapper_class=structlog.BoundLogger,
+            context_class=dict,
+            logger_factory=structlog.PrintLoggerFactory(),
+            cache_logger_on_first_use=True,
+        )
+        loggers["structlog"] = structlog.get_logger("structlog_test")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)